roles/lib_openshift/library/oc_volume.py

```diff
@@ -1750,24 +1756,28 @@ def create_volume_structure(volume_info):
         ''' return a properly structured volume '''
         volume_mount = None
         volume = {'name': volume_info['name']}
-        if volume_info['type'] == 'secret':
+        volume_type = volume_info['type'].lower()
```

Comment:
Yeah, this was my fault.  @mtnbikenc had made changes to this file to do the insensitive case checking.  I stomped his changes in the first PR so I reverted back.  That was my fault.